src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/ScalarWaveGr.cpp

```diff
@@ -0,0 +1,145 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/CurvedWaveEquation/ScalarWaveGr.hpp"
+
+#include <pup.h>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
```

Comment:
double-check these includes, you don't need a lot of them

Reply:
removed relic includes, thanks!